### PR TITLE
Contact Section, Colour/Theme Changes & Refactored Code

### PR DESCRIPTION
**Pull request by MattW-R**
Added contact section as footer
Changed highlighted text colour
Navbar colour changes
Refactored & cleaned up code
Changes for cross browser compatibility

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,25 +1,87 @@
...
 }
 
+a, h2, p, h1 {
+	margin: 10px;
+	padding: 5px;
+}
+
+a, h2 {
+	font-size: 1rem;
+	align-content: center;
```
Comment (emmatrain) [inline styles.css]: Not sure this does anything for a `<a>` element. But could be wrong and it's doing something somewhere! so hard to tell!

File: styles.css [hunk 2]
```diff
@@ -179,36 +286,59 @@ article:hover p, article:focus-within p {
 }
 
 @media screen and (max-width: 630px) {
+	html {
+		font-size: 18px;
```
Comment (emmatrain) [inline styles.css:285]: There's probs a real reason for this but you've used `rem` the whole way through  which is meant to help with responsiveness but then used px in the media query?

**Review by emmatrain — COMMENTED** [review]
Looks good to me, very few comments (not sure if that's helpful or unhelpful haha!) I checked in browser stack for all browsers plus android and iphone and looks good.

File: styles.css (hunk 2 above)
Comment (MattW-R) [reply styles.css:285]: The rems are responding to the html absolute font-size I believe

**Pull request merged**